src/rpc/mining.cpp

```diff
@@ -154,7 +154,7 @@ static UniValue generatetodescriptor(const JSONRPCRequest& request)
             {"maxtries", RPCArg::Type::NUM, /* default */ "1000000", "How many iterations to try."},
         },
         RPCResult{
-            "[ blockhashes ]     (array) hashes of blocks generated\n"},
+            "[ blockhashes ]     (json array) hashes of blocks generated\n"},
```

Comment:
👀 this is json-rpc?

Reply:
Yes, RPC arguments are called this way. Also, most of the RPC results are called this way. This adjusts the remaining ones.